assets/src/components/Line.js

```diff
@@ -17,9 +17,9 @@ function Line (props) {
   //   })
   // })
 
-  const lineHeight = labelPlacement === 'up' || labelPlacement === 'down'
+  const lineHeight = labelPlacement === 'up1' || labelPlacement === 'down1'
```

Comment:
What is the rationale behind label convention `up1` and`down1`? are there `up2` and down2`? also to me, up1 naming doesn't tell much that variable purpose? can it name be more meaningful?

Reply:
Previously the label placement was used to describe if something would go up or down and/or left or right all in one.  With that logic "up" might mean "up and to the right" while downLower might mean down more than down and left.  It all seemed confusing to me and what is lower than downLower if we need something lower than downLower.. i don't like where that was going!

Now the left/right aspect is determined by the value of the line, so only the up/down offset is really controlled by the labelPlacement.  To me 'up1' means offset up 1 "unit" and 'down2' means offset down 2 "units"

Reply:
I guess there is only `up1` and both `down` with 1 and 2

Reply:
`To me 'up1' means offset up 1 "unit" and 'down2' means offset down 2 "units"` can you add that has comments somewhere make sense to you? It will be clear for people who step on this code next

Reply:
Done